components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -716,6 +716,16 @@
 <config_ecosysTracers_diagnostic_fields_level4>.false.</config_ecosysTracers_diagnostic_fields_level4>
 <config_ecosysTracers_diagnostic_fields_level5>.false.</config_ecosysTracers_diagnostic_fields_level5>
 
+<!-- tracer_forcing_freshwaterTracers -->
+<config_use_freshwaterTracers>.false.</config_use_freshwaterTracers>
+<config_use_freshwaterTracers_surface_bulk_forcing>.false.</config_use_freshwaterTracers_surface_bulk_forcing>
```

Comment:
would it be better to set "config_use_freshwaterTracers_surface_bulk_forcing" to .true. in e3sm defaults same as in mpas-o defaults?

Reply:
Yes, I'll do that now